src/tagging/components/tag.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+ import style from './style';
+
+class Tag extends React.Component {
+  handleClick = () => this.props.onTagDeleteClick(this.props.tagCategory, this.props.tagValue);
+
+  render() {
+    return (
+      <li key={`${this.props.tagCategory}: ${this.props.tagValue}`}>
+        <span className="label label-info">
+          {this.props.tagCategory}: {this.props.tagValue}
+          <a href="#" onClick={this.handleClick}>
+            <span className="pficon pficon-close" aria-hidden="true" />
+            <span className="sr-only">Remove</span>
```

Comment:
all strings should be available as props, so later on  you can i18n them.

Reply:
Done, strings are available as props with default values.